.ci.yaml

```diff
@@ -2326,6 +2329,17 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: flutter_gallery__transition_perf_with_semantics
 
+  # MotoG4, Skia
+  - name: Linux_android new_gallery__transition_perf
```

Comment:
should this build be in this PR?

Reply:
I moved this existing build from another place in the file to be next to all the other `new_gallery__transition_perf` benchmarks.